### ISSUE TITLE
forward prediction

### ISSUE THREAD

**Issue by manand881** [msaleem18/Bitcoin-Price-Prediction-LSTM#1]
i see that the graph does not predict the price in the future. if so, what the point of this exercise?

**Comment by msaleem18** [comment]
I completely agree, it kind of goes to show that the crypto market is to complex to be simplified using just a few variables (as in this model). News, Econometrics and Big players in the market have influences that are difficult to predict for sure
